web/src/components/ModalThreeButton.tsx

```diff
@@ -0,0 +1,84 @@
+import { ModalZeroButton } from "@/components/ModalZeroButton";
+import { PrimaryButton } from "@/components/njwds-extended/PrimaryButton";
+import { SecondaryButton } from "@/components/njwds-extended/SecondaryButton";
+import { UnStyledButton } from "@/components/njwds-extended/UnStyledButton";
+import { ReverseOrderInMobile } from "@/components/njwds-layout/ReverseOrderInMobile";
+import { MediaQueries } from "@/lib/PageSizes";
+import { useMediaQuery } from "@mui/material";
+import { ReactElement, ReactNode } from "react";
+
+interface Props {
+  isOpen: boolean;
+  close: () => void;
+  title: string;
+  children: ReactNode;
+  primaryButtonText: string;
+  primaryButtonOnClick: () => void;
+  secondaryButtonText: string;
+  secondaryButtonOnClick?: () => void;
+  tertiaryButtonText: string;
+  tertiaryButtonOnClick?: () => void;
+  isLoading?: boolean;
+}
+
+export const ModalThreeButton = (props: Props): ReactElement => {
+  const isMobileAndUp = useMediaQuery(MediaQueries.mobileAndUp);
+
+  const buttonRow = (
+    <div
+      className={`padding-x-4 padding-y-3 ${
+        isMobileAndUp
+          ? "flex flex-row flex-align-center flex-justify-end"
+          : "flex flex-column flex-align-center gap-2"
+      }`}
+    >
+      <ReverseOrderInMobile
+        className={`flex flex-column flex-align-stretch ${isMobileAndUp ? "" : "gap-2"}`}
+      >
+        <>
+          <UnStyledButton
+            onClick={props.tertiaryButtonOnClick ?? props.close}
+            dataTestid="modal-button-tertiary"
+            className={`${isMobileAndUp ? "padding-right-2" : ""}`}
+          >
+            <span className="text-bold" style={{ color: "#538200" }}>
```

Comment:
2 different notes:
- `UnStyledButton` has a `isTextBold` prop. It looks like setting that would have the same outcome as applying the class name here.
- I only see that hex code 1 other place in our codebase (in a commit from 4 years ago!). If we intend to start using this color, we might want to more formally incorporate this into our styles.

Reply:
It's funny because it's called... unstyled button... 🤔